test/verify/check-machines-dbus

```diff
@@ -58,6 +58,32 @@ class TestMachinesDBus(machineslib.TestMachines):
         if self.machine.image == "ubuntu-stable":
             self.machine.execute("chmod o+rwx /run/libvirt/libvirt-sock")
 
+    def testAutostart(self):
+        b = self.browser
+        m = self.machine
+
+        self.startVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        b.click("tbody tr th") # click on the row header
+        b.wait_present("#vm-subVmTest1-state")
+        b.wait_in_text("#vm-subVmTest1-state", "running")
+
+        b.wait_present("#vm-subVmTest1-autostart-checkbox")
```

Comment:
This wouldn't be true on OSes that don't yet have libvirt-dbus. On these it should instead check wait_not_present and return from the test early.

Reply:
Ah, this is check-machines-dbus, so this test case is fine actually. But could you please add a corresponding short test  to check-machines-virsh that it's not visible?

Reply:
Done